config-model/src/test/java/com/yahoo/schema/processing/IndexingInputsTestCase.java

```diff
@@ -6,43 +6,104 @@
 import com.yahoo.yolean.Exceptions;
 import org.junit.jupiter.api.Test;
 
-import java.io.IOException;
-
-import static com.yahoo.schema.processing.AssertSearchBuilder.assertBuildFails;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
 
 /**
  * @author Simon Thoresen Hult
+ * @author bratseth
  */
 public class IndexingInputsTestCase {
 
     @Test
-    void requireThatExtraFieldInputExtraFieldThrows() throws IOException, ParseException {
-        assertBuildFails("src/test/examples/indexing_extra_field_input_extra_field.sd",
-                "For schema 'indexing_extra_field_input_extra_field', field 'bar': Indexing script refers " +
-                        "to field 'bar' which is neither a field in document type " +
-                        "'indexing_extra_field_input_extra_field' nor a mutable attribute");
+    void requireThatExtraFieldInputExtraFieldThrows() throws ParseException {
```

Comment:
Agreed at least for those tests that have many of them (it doesn't save much). They are decoupled and standalone now, which is what I wanted to achieve, so that can be done later if somebody visits this area again.